### ISSUE TITLE
Vercel Authentication error when creating new project

### ISSUE THREAD

**Issue by cungminh2710** [vercel/terraform-provider-vercel#142]
When running apply on a new project without the optional [vercel_authentication](https://registry.terraform.io/providers/vercel/vercel/latest/docs/resources/project#vercel_authentication), the error pops up:

```
"provider[\"registry.terraform.io/vercel/vercel\"]" produced an unexpected
new value: .vercel_authentication: was null, but now
cty.ObjectVal(map[string]cty.Value{"protect_production":cty.False}).
```

**Comment by dglsparsons** [comment]
Hey @cungminh2710, thanks for raising this. I'll get looking into it and get a fix out ASAP. 🙏

**Comment by dglsparsons** [comment]
Apologies for the slow response on this one. 

Turns out the change in behaviour is due to Vercel Authentication being made the default for new projects -> https://vercel.com/changelog/deployment-protection-is-now-enabled-by-default-for-new-projects

Fix for this is incoming soon!

**Comment by dglsparsons** [comment]
This is fixed in v0.16.0
